### PR TITLE
feat: web app test server plugin

### PR DESCRIPTION
**Pull request by seebergs**
## Changes

Initial version

## Test

See README

### CODE REVIEW

File: python/plugins/web-app-test-server/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# sd-web-app-test-server
```
Comment (hdd) [inline python/plugins/web-app-test-server/README.md]: update name to remove sd prefix (not needed in recipes)

File: python/plugins/web-app-test-server/hook/plugin.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+def start_wsgi_server_thread():
+    logger = logging.getLogger()
+    logger.info('Starting server...')
+    thread = threading.Thread(target=start_server)
+    thread.daemon = True
+    thread.start()
+
+def register(session, **kw):
+    if not isinstance(session, ftrack_api.session.Session):
+        return
```
Comment (hdd) [inline python/plugins/web-app-test-server/hook/plugin.py:42]: shall we log.error something here in case something goes wrong ?

**Review by hdd — CHANGES_REQUESTED** [review]

File: python/plugins/web-app-test-server/hook/plugin.py (hunk 2 above)
Comment (seebergs) [reply python/plugins/web-app-test-server/hook/plugin.py:42]: Good idea, added.

File: python/plugins/web-app-test-server/README.md (hunk 1 above)
Comment (seebergs) [reply python/plugins/web-app-test-server/README.md]: Removed prefix.

**Review by hdd — APPROVED** [review]

**Pull request merged**